web/tests/functional/diff_local_remote/test_diff_local_remote.py

```diff
@@ -207,14 +207,14 @@ def test_local_cmp_filter_unres_severity(self):
     def test_local_cmp_filter_unres_filepath(self):
         """Filter unresolved results by file path."""
         res = self.get_local_remote_diff(['--file', '*divide_zero.cpp'])
-        self.assertEqual(len(re.findall(r'divide_zero.cpp', res)), 4)
-        self.assertEqual(len(re.findall(r'new_delete.cpp', res)), 0)
+        self.assertEqual(len(re.findall(r'/divide_zero.cpp', res)), 4)
```

Comment:
why do we need these changes here, because of the summary printing the results twice?

Reply:
Yes, you are right, because of the summary.

```sh
[INFO 2019-06-06 09:10] - Matching runs: simple
[MEDIUM] /home/username/projects/simple/main.cpp:3:3: expression result unused [clang-diagnostic-unused-value]
  sizeof(42);


[HIGH] /home/username/projects/simple/main.cpp:3:3: suspicious usage of 'sizeof(K)'; did you mean 'K'? [bugprone-sizeof-expression]
  sizeof(42);



----==== Summary ====----
-----------------------
Filename | Report count
-----------------------
main.cpp |            2
-----------------------
-----------------------
Severity | Report count
-----------------------
HIGH     |            1
MEDIUM   |            1
-----------------------
----=================----
Total number of reports: 2
----=================----

[INFO 2019-06-06 09:10] - Compared local report directory /home/username/projects/simple/reports and remote run(s) simple (matching: simple).
```

Instead of this I change the expected result count. This way we test the summary too.